dcicutils/bundle_utils.py

```diff
@@ -657,6 +658,7 @@ def load_items(filename: str, tab_name: Optional[str] = None, escaping: Optional
     if validate:
         problems = validate_data_against_schemas(checked_items, portal_env=portal_env, portal_vapp=portal_vapp,
                                                  override_schemas=override_schemas)
+        return checked_items, problems
         error_summary = summary_of_data_validation_errors(problems)
```

Comment:
Is this return statement intended? Dead code beneath it?

Reply:
actual dead code - deleted